src/classes/AffiliationMappingsHealthCheckVMapper.cls

```diff
@@ -512,7 +512,7 @@ public virtual with sharing class AffiliationMappingsHealthCheckVMapper {
                 Label.stgHCAfflMapContactPrimaryFieldValid, 
                 new List<Object>{
                     affiliationMappingsModel.primaryAffiliationField, 
-                    affiliationMappingsModel.accountRecordTypeFieldValue
+                    accountlookupFieldOnContactFieldModel.name
```

Comment:
Very minor one. ‘l’ should be capitalized. Not a big deal atm thou.

Reply:
Done